application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/views/stacktrace/StacktraceView.java

```diff
@@ -458,7 +467,7 @@ private void rebuildViewer() {
 			Display.getCurrent().asyncExec(() -> {
 				if (!viewer.getControl().isDisposed()) {
 					setViewerInput(oldInput);
-					if (reducedTree && oldInput != null) {
+					if (!reducedTree && oldInput != null) {
```

Comment:
Don't get this. If we don't want to reduce - then reduce? Very strange behaviour.

Reply:
seem I've forgotten to revert as I've been playing a bit more with it